examples/README.md

```diff
@@ -0,0 +1,35 @@
+# Examples
+
+All of these examples use a configurable Acorn Agent Server docker image.
+
+## Custom API Examples
+
+You can run the following examples in Docker by standing up an API and running Acorn Agent as a server in front of it:
+
+* [Customer Credit Card](https://github.com/DataSQRL/datasqrl-examples/tree/main/finance-credit-card-chatbot): Builds a ChatBot that analyzes a customer's credit card transactions to answer questions about spending.
+* [IoT Bot](https://github.com/DataSQRL/datasqrl-examples/tree/main/iot-sensor-metrics): A data agent for retrieving and analyzing metrics in realtime.
+* [Credit Card Rewards](https://github.com/DataSQRL/datasqrl-examples/tree/main/finance-credit-card-chatbot): An agent that helps customers pick the best credit card to maximize rewards.
+
+For these examples, you will stand up an API locally that Acorn Agents instruments for LLM usage. Hence, these examples are a little more involved to run than the public API examples below. However, because we have control over the API, we can use more of Acorn Agent's feature set like storing and retrieving user messages, adding more complex tools for data retrieval, and custom data visualization.
+
+These examples show you what building a production-grade application with Acorn Agent looks like. 
```

Comment:
@mbroecheler But these examples no longer live in this repo. Do you plan to include a link to the datasqrl-examples repo or to move them back in here?

Reply:
I added a link.